service/history/shard/context.go

```diff
@@ -18,6 +18,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+//go:generate mockgen -package $GOPACKAGE -source $GOFILE -destination interface_mock.go -package shard github.com/uber/cadence/history/shard/context Context
```

Comment:
I'd rename interface_mock.go to context_mock.go for co-locating these and make relation obvious when looking at the file tree